sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClient.java

```diff
@@ -295,9 +296,10 @@ public ConfigurationSetting deleteSetting(String key, String label) {
      * @throws HttpResponseException If {@link ConfigurationSetting#getKey() key} is an empty string.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<ConfigurationSetting> deleteSettingWithResponse(ConfigurationSetting setting, boolean ifUnchanged,
```

Comment:
`etag` to `Etag`, [here](https://github.com/Azure/azure-sdk-for-java/pull/5978/files#diff-c0146ef445b1db25306d811380105454L288) and [here](https://github.com/Azure/azure-sdk-for-java/pull/5978/files#diff-c0146ef445b1db25306d811380105454L294)